### PR TITLE
ERSSUP-59667 - Policy to translate nhsd-ers-obo-user-id header to x-ers-obo-user-id

### PR DESCRIPTION
**Pull request by nhsd-nigel-mellor**
… x-ers-obo-user-id]-[NM]

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: proxies/live/apiproxy/targets/ers-target.xml [hunk 1]
```diff
@@ -1,120 +1,119 @@
...
+                <Condition>(response.header.x_ers_transaction_id ~~ ".+")</Condition>
+            </Step>
+            <Step>
+                <Name>AssignMessage.Remove.nhsd-correlation-id-header</Name>
+                <Condition>(response.header.nhsd-correlation-id ~~ ".+")</Condition>
+            </Step>
+        </Response>
+    </PreFlow>
+    <HTTPTargetConnection>
+        <SSLInfo>{% if '--ft-' in (ERS_TARGET_SERVER | default('e-referrals-service-api')) %} <TrustStore>ref://e-referral-service-ca</TrustStore> {% endif %} <Enabled>true</Enabled></SSLInfo>
```
Comment (jamiegreen4) [inline proxies/live/apiproxy/targets/ers-target.xml:95]: Did you mean to change the formatting of the if/endif here?

**Review by nhsd-jack-wainwright — DISMISSED** [review]
Looks good to me 🙂

File: proxies/live/apiproxy/policies/AssignMessage.Swap.nhsd-ers-obo-user-id.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.Swap.nhsd-ers-obo-user-id">
+    <Set>
+        <Headers>
+            <Header name="x-ers-obo-user-id">{message.header.nhsd-ers-obo-user-id}</Header>
+        </Headers>
```
Comment (andrewclark2020) [inline proxies/live/apiproxy/policies/AssignMessage.Swap.nhsd-ers-obo-user-id.xml:6]: @nhsd-nigel-mellor  tests have now been added to test the headers. You could easily add these headers to the test and that will ensure the values are copied over.

**Pull request closed without merging**